sdks/java/io/amazon-web-services/src/test/java/org/apache/beam/sdk/io/aws/sqs/SqsIOTest.java

```diff
@@ -117,7 +117,6 @@ protected void before() {
                   new AWSStaticCredentialsProvider(new BasicAWSCredentials(accessKey, secretKey)))
               .withEndpointConfiguration(
                   new AwsClientBuilder.EndpointConfiguration(endpoint, region))
-              .withRegion(region)
```

Comment:
Why `withRegion` was dropped? Because region is used in `EndpointConfiguration` creation?

Reply:
Yes and now the amazon sdk validates this so the test failed.